CMSgov/mint-app
# Easi 2308/ CR and TDL

# EASI-2308

## Changes and Description

- CR/TDL Task list banner
- CR/TDL view with table of all model plan CRTDLs
- Add/Edit form
- Remove CRTDL capability
- Cypress/unit testing


![Screen Shot 2022-09-21 at 3 06 50 PM](https://user-images.githubusercontent.com/95709965/191589569-f0b48c2c-fb7f-41f2-8ea6-e48bd7aa43b1.png)

![Screen Shot 2022-09-21 at 3 07 09 PM](https://user-images.githubusercontent.com/95709965/191589629-2ad78850-f29f-452b-8415-c85ab7f91529.png)

![Screen Shot 2022-09-21 at 3 07 24 PM](https://user-images.githubusercontent.com/95709965/191589678-b4f78d99-fa00-4ba2-af25-0939cd9babe8.png)

## Notes

 - Refactored task list document banner as well for standardization

## How to test this change

- Seed database, click on 'Plan With CRs and TDLs'
- Click on 'Manage CRs and TDLs'
- Verify exising CR/TDLs populate table
- Edit one
- Remove one
- Add one

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
